src/components/ComputeModal.js

```diff
@@ -90,7 +90,7 @@ const DataprocDiskSelector = ({ value, onChange }) => {
       label({ htmlFor: id, style: styles.label }, ['Disk size (GB)']),
       h(NumberInput, {
         id,
-        min: 60, // less than this size causes failures in cluster creation
+        min: 80, // less than this size causes failures in cluster creation
```

Comment:
is this for boot disk or user disk?

Reply:
This is dataproc so they are the same disk.. the custom image is generated with a 80g disk too.

Reply:
ah right